//! THIS IS A TEST PROGRAM
//! RUNTIME
//! FAILURE
//! CODE: 1337

func foo() -> u32 {
    return 1;
}

func main() -> u32 {
    if (foo() == 1) {
        return 1337;
    }
    return 0;
}